### DISCUSSION TITLE
i have to many "Unfortunately" mail on my student mail

### DISCUSSION THREAD

**Discussion by cin0224** [community/community#51324] [GitHub Education]
hi i'm from korea(on south) and I'm a university student. 
btw i want git hub education for git and copilot.
I always get rejected by requests Maybe more than 20 times.
with 
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on mail

I've tried it many times Through my cell phone, through my laptop, through my desktop, through my cell phone camera, my school PDF certification, my student ID, my enrollment certificate, which is now completely printed on paper, but the whole process has been rejected and I am in despair

The problem was different every time.
It's so sad to be rejected by automatic response, saying that I can't confirm the name of the school (my university didn't write a consistent name in the English document)
The screen didn't look blurry, show more information on the github account (I even drew with a drawing board because of this For my profile) I can no longer obtain a certificate in English from the school What should I do?

Why don't you accept PDFs that can't be operated?
Why don't you accept the image file that you already have because of the possibility of manipulation?

Why is the picture uploaded with resized quality when I upload it using my phone? The environment is an iPhone 12pro ,ios 15.4 version Safari web browser environment.

My laptop is a high-end laptop that costs more than 2000 dollars, but it is difficult to secure a proper picture because official documents in Korea are printed vertically

Improve email or photo recognition algorithms for help For non-English cultures, getting certification is too hard

**Comment by samus-aran** [comment]
Hi @cin0224 👋 

We're happy you're here.

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient especially when applying using non English evidence or English is not your first language.

How did you get on? What was your last rejection reasons and perhaps we can assist you from there ⭐
